mne/io/edf/edf.py

```diff
@@ -638,7 +638,10 @@ def _read_edf_header(fname, exclude, infer_types, include=None):
 
     with open(fname, 'rb') as fid:
 
-        fid.read(8)  # version (unused here)
+        version = fid.read(8).decode('latin-1').rstrip()
```

Comment:
According to the specs, this must be ASCII encoded. I know we've already sprinkled in Latin-1, because it allows us to read slightly non-standard files, but at least in the version I think it should be ASCII.